build.yaml

```diff
@@ -882,30 +882,23 @@ steps:
       - from: /wheel-container.tar
         to: /io/hail/wheel-container.tar
     dependsOn:
-      - base_image
+      - service_base_image
       - build_hail
       - merge_code
-  - kind: deploy
-    name: deploy_hail_repl
-    runIfRequested: true
-    scopes:
-      - dev
-    namespace:
-      valueFrom: default_ns.name
-    config: hail/hail_repl.yaml
-    dependsOn:
-      - default_ns
-      - hail_repl_image
   - kind: runImage
-    name: upload_test_resources_to_gcs
+    name: upload_test_resources_to_blob_storage
     image:
-      valueFrom: base_image.image
+      valueFrom: hail_pip_installed_image.image
     script: |
       set -ex
       cd /io/repo/hail/
-      gcloud -q auth activate-service-account --key-file=/test-gsa-key/key.json
-      gsutil -m cp -r src/test/resources/* gs://{{ global.hail_test_gcs_bucket }}/{{ token }}/test/resources/
-      gsutil -m cp -r python/hail/docs/data/* gs://{{ global.hail_test_gcs_bucket }}/{{ token }}/doctest/data/
+      export GOOGLE_APPLICATION_CREDENTIALS=/test-gsa-key/key.json
```

Comment:
Will this always use `GOOGLE_APPLICATION_CREDENTIALS` or will it `AZURE_APPLICATION_CREDENTIALS`?

Reply:
Ah, OK, from Zulip I see we need to set both:
```
export GOOGLE_APPLICATION_CREDENTIALS=/test-gsa-key/key.json
export AZURE_APPLICATION_CREDENTIALS =/test-gsa-key/key.json
```
even though that looks weird. We'll never try to use the google ones.